server/node_test.go

```diff
@@ -121,6 +121,7 @@ func TestBootstrapCluster(t *testing.T) {
 	for _, kv := range rows {
 		keys = append(keys, kv.Key)
 	}
+	// TODO(marc): this depends on the sql system objects.
```

Comment:
Either ignore them, or have a little helper somewhere. Still not sure and definitely not critical, hence the TODO. At the very least, it won't surprise anyone else.